src/test/java/co/rsk/bitcoinj/script/FastBridgeRedeemScriptParserTest.java

```diff
@@ -82,20 +82,6 @@ public void createMultiSigFastBridgeRedeemScript_zero_hash_as_derivation_argumen
         FastBridgeRedeemScriptParser.createMultiSigFastBridgeRedeemScript(redeemScript, Sha256Hash.ZERO_HASH);
     }
 
-    @Test
```

Comment:
Changes on the FastBridge classes are not necessary since they will be removed in another PR.

Reply:
This test was failing. This is why I removed it. So you don't need to worry. It is just this tiny change.